myconfig/repo-config.csv

```diff
@@ -0,0 +1,2 @@
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List
```

Comment:
Why this change?

Reply:
Seems unrelated?

Reply:
Sorry some error occured. This folder is supposed to be ignored